aviary/validation_cases/benchmark_utils.py

```diff
@@ -5,7 +5,7 @@
 from aviary.utils.test_utils.assert_utils import warn_timeseries_near_equal
 
 
-def compare_against_expected_values(prob, expected_dict):
+def compare_against_expected_values(prob, expected_dict, simple_flag=False):
```

Comment:
If this is temporary until the distinction between simple and FLOPS is removed, this is fine, but if it is more permanent I think it should be something more descriptive like use_simplified_mission

Reply:
it is temporary